app/models/allocation.rb

```diff
@@ -85,10 +85,12 @@ def allocate_all_claims_or_none!(claims)
 
   def allocate_or_error_claim!(claim)
     if claim.case_workers.exists?
-      errors.add(:base,"Claim #{claim.case_number} has already been allocated to #{claim.case_workers.first.name}")
+      errors.add(:base, "Claim #{claim.case_number} has already been allocated to #{claim.case_workers.first.name}")
     else
       allocate_claim! claim
     end
+  rescue => ex
```

Comment:
Consider specifying the class of exceptions we're looking out for. See http://stackoverflow.com/questions/10048173/why-is-it-bad-style-to-rescue-exception-e-in-ruby